### DISCUSSION TITLE
Streaming Netflix and others without OMR-bypass?

### DISCUSSION THREAD

**Discussion by mathieuric** [Ysurac/openmptcprouter#2638] [Q&A]
Is there a way to access streaming services like Netflix and Disney+ without OMR-bypass?
I would really like to use the full power of my aggregated connections to watch 4K movies (legally).

The only thing I’ve tried so far, without success, is a different VPS provider because they had a data center in my country.

I’ve read a little about VPS’ and VPNs  with dedicated IPs and it looks like a  solution but I’m not sure…

What would be needed to achieve this?

Thanks.

**Comment by iambenmitchell** [comment]
You either need to find a niche hosting provider that isn't known to Netflix yet, or you can run a Netflix compatible VPN from a provider like Windscribe on the VPS but this is complicated and I haven't managed to get this working myself yet. 

An easier way would be to run the VPN on your client instead.. or get a hardware VPN and use OMR as the WAN

**Comment by mathieuric** [comment]
@iambenmitchell Thanks for the reply. It worked by connecting the clients to the VPN as you suggested (I am using NordVPN).

I did a little more research and found this article showing how to connect an [OpenVPN client using LuCI](https://openwrt.org/docs/guide-user/services/vpn/openvpn/client-luci). I followed it and... magic:magic_wand:! I can now access any streaming service from a device connected to the OpenMPTCPRouter without the need to install a VPN client application.

For anyone interested, the NordVPN config files for OpenVPN can be found [here](https://nordvpn.com/ovpn/). Just make sure to use one for the region you want to connect to.
